.travis.yml

```diff
@@ -13,4 +13,6 @@ notifications:
   email:
     on_success: never
     on_failure: always
-    
\ No newline at end of file
+    
+
+dist: precise
```

Comment:
Since it looks like removing 'package-lock.json' fixed the issue - can you revert this change to .travis.yml?